lib/fluent/command/plugin_config_formatter.rb

```diff
@@ -162,6 +162,11 @@ def dump_section_markdown(base_section, level = 0)
     else
       sections, params = base_section.partition {|_name, value| value[:section] }
     end
+    if @compact and not params.empty?
+      dumped << "### Configuration\n\n"
+      dumped << "|parameter|type|description|default|\n"
+      dumped << "|---|---|---|---|\n"
+    end
```

Comment:
I want to add a separated instance variable to handle markdown table....

Reply:
It is reasonable, I'll fix.